app/components/drawer_layout_polyfill.js

```diff
@@ -0,0 +1,401 @@
+import React, { Component } from 'react';
+import {
+    Animated,
+    Dimensions,
+    Keyboard,
+    PanResponder,
+    StyleSheet,
+    TouchableWithoutFeedback,
+    View,
+    I18nManager,
+} from 'react-native';
+
+const MIN_SWIPE_DISTANCE = 3;
+const DEVICE_WIDTH = parseFloat(Dimensions.get('window').width);
```

Comment:
this won't recalculate on device rotation

Reply:
let me know if it's causing actual issues on rotate because I can't test right now